openupgrade_scripts/scripts/account/14.0.1.1/post-migration.py

```diff
@@ -605,6 +605,69 @@ def fill_account_journal_payment_credit_debit_account_id(env):
         )
 
 
+def create_new_counterpar_account_payment_transfer(env):
+    # Create new counterpart payment with account payment transfer
+    openupgrade.logged_query(
+        env.cr,
+        """
+        INSERT INTO account_payment (move_id, is_internal_transfer, partner_type,
+            payment_type,
+            amount, currency_id,
+            destination_account_id, partner_id, journal_id,
+            create_uid, create_date, write_uid, write_date)
+        SELECT move.id, true, ap.partner_type,
+            CASE
+            WHEN liquidity_line.amount_currency > 0 THEN 'inbound' ELSE 'outbound'
+            END,
+            ABS(liquidity_line.amount_currency), liquidity_line.currency_id,
+            ap.destination_account_id, ap.partner_id, move.journal_id,
+            move.create_uid, move.create_date, move.write_uid, move.write_date
```

Comment:
Better to use the same dates as the original payment of type transfer:

```suggestion
            ap.create_uid, ap.create_date, ap.write_uid, ap.write_date
```

Reply:
Thank you, I'll fix it